### ISSUE TITLE
Explain CubicFlatteningIter tolerance

### ISSUE THREAD

**Issue by dbrgn** [nical/lyon#18]
Hi. Great library, thanks a lot! :)

I used the Bézier part to do flattening of SVG paths. Could you maybe explain the tolerance value in the documentation in laymans terms? From my experiments, the lower the value the better the approximation. But what does the value actually mean? Does it have to do with the angle between two line segments?

(I should probably read the paper, but adding a short explanation to the docs would also help other people :))

**Comment by nical** [comment]
Hi, thanks for the compliment! And thanks for filing the other issue, and sorry for not noticing it earlier.
The flattening tolerance is the maximum distance between the theoretical curve and the flattened approximation for an infinitely small line width. Ideally the this tolerance should take the curve width into account, but it does not, so if you want to be strictly correct you need to take it into account. I'll update the docs.

**Comment by nical** [comment]
Fixed in #22.
